man/setattr.Rd

```diff
@@ -24,6 +26,8 @@ setnames(x,old,new,skip_absent=FALSE)
 
   \code{setattr} is a more general function that allows setting of any attribute to an object \emph{by reference}.
 
+  \code{setlevels} is a function to set the levels of factor vector.
```

Comment:
I feel this isn't needed as `Details`. If anything in Details, we should note the sort of consistency checks that are being run that make it advantageous vs `setattr`